UKHomeOffice/sgmr-service
# NMSW-108 Add help page image

### AC / Description of changes

GIVEN I am viewing the help page from the footer
WHEN I scroll to the bottom of the page
THEN I see the map 

Both help pages contain a new region image, shown below the phone number section

### To Test

- Run the app locally
- Go to localhost:8080
- Sign in
- Click on the help in the navigation
- Scroll to below the phone numbers
- > There should be an image showing the boundary map
- Go to /voyage plans
- Click on 'Start a new voyage plan'
- Scroll to below the phone numbers
- The image should be present here too

### Notes

* I have added a `__Mocks__` folder containing a file called `assetMock.js` as the tests were throwing errors without this - see [Using Jest with Webpack](https://jestjs.io/docs/webpack#handling-static-assets) - (May need to come back to this in the future if there is an issue)

---

**Remember to merge to feature branches not `master`.**
